homeassistant/bootstrap.py

```diff
@@ -86,13 +86,12 @@ def from_config_dict(config: Dict[str, Any],
                              log_no_color)
 
     core_config = config.get(core.DOMAIN, {})
-    has_api_password = bool((config.get('http') or {}).get('api_password'))
-    has_trusted_networks = bool((config.get('http') or {})
-                                .get('trusted_networks'))
+    has_api_password = bool(config.get('http', {}).get('api_password'))
+    trusted_networks = config.get('http', {}).get('trusted_networks')
```

Comment:
The reason we had an `or` is because people can have a config like this:

```yaml
http:
```

And then it can be None? Although if I remember correctly, we do replace `None` with `{}`…

Reply:
`trusted_networks` will be `None` for the case
```yaml
http:
```